layouts/partials/scientific-domains.html

```diff
@@ -100,7 +100,7 @@
             <td class="lastrow-center-text"><a href="http://etetoolkit.org/">ETE</a></td>
             <td class="lastrow-center-text"><a href="https://emcee.readthedocs.io/">emcee</a></td>
             <td class="lastrow-center-text"><a href="https://fenicsproject.org/">FEniCS</a></td>
-            <td class="lastrow-center-text"></td>
+            <td class="lastrow-center-text"><a href="https://github.com/pybamm-team/PyBaMM/">PyBaMM</a></td>
```

Comment:
```suggestion
            <td class="lastrow-center-text"><a href="https://www.pybamm.org">PyBaMM</a></td>
```
@MattiaAldighieri, thank you for working on this PR! The convention for this section of the website is adding a link to a project's website.

Reply:
Oh ok, I hadn't noticed.